release.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+function help() {
+    echo "Usage: sh release.sh <RELEASE_VERSION>"
+    echo ""
+    echo "Parameters:"
+    echo "  RELEASE_VERSION: SemVer Version number for the release"
+}
+
+semver_pattern="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)$"
+
+RELEASE_VERSION=${1}
+
+[[ -z "${RELEASE_VERSION}" ]] && echo "ERROR: RELEASE_VERSION is empty" && help  && exit 1
+[[ ! "${RELEASE_VERSION}" =~ $semver_pattern ]] && echo "ERROR: RELEASE_VERSION does not match the SemVer specification" && help && exit 1
+
+printf "\n%s\n" "Create new branch release/${RELEASE_VERSION}"
+git checkout -b release/${RELEASE_VERSION}
+
+printf "\n%s\n"  "Change Version in package.json"
+npm version ${RELEASE_VERSION} --no-git-tag-version
+
+printf "\n%s\n" "Run npm i"
+npm i
+
+printf "\n%s\n"  "Run npm audit fix"
+npm audit fix
+
+printf "\n\n%s "  "Would you like to commit and push these changes? (y/n)"
```

Comment:
The release process mentions to update the changelog.md.  Maybe a hint would be a good option.